rssfeed/tests/settings/19.py

```diff
@@ -0,0 +1,63 @@
+import os
+import glob
+from os.path import expanduser
+
+
+BASE_DIR = os.path.join(
+    glob.glob(os.environ["VIRTUAL_ENV"] +  "/lib/*/site-packages")[0],
+    "spliceworks"
+)
+
+DEBUG = True
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.sqlite3",
```

Comment:
Just check the engine to use. I am a bit out of date on our newest testing philosophies, but heard along the line all tests should use postgres.